src/readline.c

```diff
@@ -47,10 +47,10 @@ static char	*get_inline_shell_display(void)
 
 	getcwd(cwd, 1024);
 	path = get_last_slash_arg(cwd);
-	tmp = ft_strdup("\r\033[1;31m[user@minishell ");
+	tmp = ft_strjoin(ft_strdup(getenv("USER")), "\033[0;31m@minishell ");
```

Comment:
Careful as `strdup` and `strjoin` both allocate memory and should be freed separately

Reply:
We are not manipulating the environment directly, we are using our own wrapper around it. And for that we use `ms_getenv` insted of the proper system `getenv` itself.